src/containerapp/azext_containerapp/tests/latest/test_containerapp_azurefile.py

```diff
@@ -0,0 +1,165 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import time
+import unittest
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only, StorageAccountPreparer)
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+from .common import TEST_LOCATION
+from .utils import create_containerapp_env
+from azext_containerapp.tests.latest.common import (
+    write_test_file,
+    clean_up_test_file,
+    TEST_DIR, TEST_LOCATION)
+
+class ContainerAppMountAzureFileTest(ScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location=TEST_LOCATION)
+    def test_container_app_mount_azurefile_e2e(self, resource_group):
+        env = self.create_random_name(prefix='env', length=24)
```

Comment:
Please add following code to support test run in different locations:
`self.cmd('configure --defaults location={}'.format(TEST_LOCATION))`